.github/workflows/lint-test.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         run: cockroach quit --insecure --host=localhost:26257
 
       - name: Upload coverage report
-        uses: codecov/codecov-action@v4
+        uses: codecov/codecov-action@v3
```

Comment:
I ended up having to revert that change to v4, because CI was complaining that it couldn't find v4 anymore. Even after restarting the job. Not sure what that's about.

Reply:
Looks like the v4 tag has been removed, and had been created by mistake: https://github.com/codecov/codecov-action/issues/1089